dotnet/src/Connectors/Connectors.UnitTests/OpenAI/ChatCompletion/AzureOpenAIChatCompletionServiceTests.cs

```diff
@@ -603,12 +609,8 @@ public async Task GetChatMessageContentsUsesPromptAndSettingsCorrectlyAsync()
         Assert.Equal("This is test system message", messages[0].GetProperty("content").GetString());
         Assert.Equal("system", messages[0].GetProperty("role").GetString());
 
+        Assert.Equal("This is test prompt", messages[1].GetProperty("content").GetString());
         Assert.Equal("user", messages[1].GetProperty("role").GetString());
-
```

Comment:
Please add a unit test that verifies the content is of an array type and that each element of the array is properly initialized. Please do the same for OpenAI chat completion in the OpenAIChatCompletionServiceTests class.

Reply:
done